CHANGELOG.md

```diff
@@ -23,7 +48,15 @@ In Teleport 15, session recordings are streamed from the auth server, allowing
 playback to start before the entire session is downloaded and unpacked.
 
 Additionally, `tsh play` now supports a `--speed` flag for adjusting the
-playback speed.
+playback speed, and desktop session playback now supports seeking to arbitrary
+positions in the recording.
+
+#### Teleport Connect improvements
+
+Teleport Connect will now prompt for an MFA tap prior to accessing Kubernetes
+clusters when per-session MFA is enabled.
+
+Additionally, Teleport Connect includes support for TCP and web applications.
```

Comment:
We also added support for AWS and SAML apps by launching them in the browser (so we support all apps that Web UI supports). Do we want to mention them?

Reply:
I've made this change in the Google doc we're working in. Which will make its way back here once it's finalized. Thanks!